### DISCUSSION TITLE
[el-dropdown]使用基础用法时，触发的元素存在:focus-visible的outline未清理干净的问题

### DISCUSSION THREAD

**Discussion by wmt-gitgub** [element-plus/element-plus#17653] [Q&A]
[el-dropdown]使用基础用法时，触发的元素存在:focus-visible的outline未清理干净的问题

![微信图片_20240725095120](https://github.com/user-attachments/assets/b0dd9caa-ba57-4e01-9a5c-fd89214e4058)

**Comment by warmthsea** [comment]
https://github.com/element-plus/element-plus/issues/17388#issuecomment-2204847447

**Comment by kiinlam** [comment]
补一刀：el-dropdown 使用 hover 触发时，会导致焦点改变，比如我现在正在编辑文本框的内容，在文本框触发 blur 事件时会保存内容。此时如果鼠标移过下拉框元素时，就会触发 blur 事件，焦点被下拉框夺走了，这个体验很不好，有解决方案吗？

**Reply by Fuphoenixes** [reply]
你可以提个issue，并提供一个案例看看
